### PR TITLE
Zee validation tools for legacy Run2

### PR DESCRIPTION
**Pull request by threiten**
Moved tools for tagAndProbe to validation, added minimal configuration to run tnp including trigger matching, eleID and scales and smearings

### CODE REVIEW

File: Validation/python/tagAndProbeDumperConfig.py [hunk 1]
```diff
@@ -0,0 +1,199 @@
+import re
```
Comment (simonepigazzini) [inline Validation/python/tagAndProbeDumperConfig.py:1]: This files contains way more than it's needed to run a clean t&p, can you please rearrange it?

File: Systematics/python/flashggDiPhotonSystematics2018_cfi.py [hunk 2]
```diff
@@ -5,312 +5,488 @@
 photonScaleUncertBins = cms.PSet()
```
Comment (simonepigazzini) [inline Systematics/python/flashggDiPhotonSystematics2018_cfi.py]: It looks like this files was edited by accident. In order to avoid injecting a large, unnecessary, diff into the history please revert back the changes and only commit the relevant ones.

File: MetaData/data/Era2017_RR-31Mar2018_v1/datasets_3.json [hunk 3]
```diff
@@ -1,8 +1,4 @@
 {
```
Comment (shervin86) [inline MetaData/data/Era2017_RR-31Mar2018_v1/datasets_3.json:1]: Changes in this file should not be part of this PR. Changes here can be reverted

File: MetaData/data/cross_sections.json [hunk 4]
```diff
@@ -61,7 +61,8 @@
 	"DYToEE_NNPDF30_13TeV-powheg-pythia8"                           : { "xs" : 2008.33 },
         "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-amcatnloFXFX-pythia8"       : { "xs" : 6025.0, "itype":20                    },
 	"DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-pythia8"        : { "xs" : 6025.0, "itype":20                    },
-	"DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-herwigpp_30M"   : { "xs" : 6025.0, "itype":20                    },
+    "DYJetsToLL_M-50_TuneCUETP8M1_13TeV-madgraphMLM-herwigpp_30M"   : { "xs" : 6025.0, "itype":20                    },
+    "DYJetsToLL_M-50_TuneCP5_13TeV-madgraphMLM-pythia8" : {"xs" : 6225.42, "itype":20 },
```
Comment (shervin86) [inline MetaData/data/cross_sections.json:65]: Changes in this file should not be part of this PR. Changes here can be reverted

File: Systematics/python/flashggDiPhotonSystematics2018_cfi.py [hunk 5]
```diff
@@ -386,7 +386,7 @@
 )
 
 
-scalesAndSmearingsPrefix = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2018_Step2Closure_CoarseEtaR9Gain")
+scalesAndSmearingsPrefix = cms.string("EgammaAnalysis/ElectronTools/data/ScalesSmearings/Run2018_Step2Closure_CoarseEtaR9Gain_v2")
```
Comment (shervin86) [inline Systematics/python/flashggDiPhotonSystematics2018_cfi.py:389]: Changes in this file should not be part of this PR. Changes here can be reverted

File: Validation/plugins/FlashggTagAndProbeProducer.cc [hunk 6]
```diff
@@ -173,7 +173,6 @@ namespace flashgg {
                 tnpColl_->push_back(cand);
             }
         }
-
```
Comment (shervin86) [inline Validation/plugins/FlashggTagAndProbeProducer.cc:176]: To be reverted

File: MetaData/data/Era2017_RR-31Mar2018_v1/datasets_3.json (hunk 3 above)
Comment (simonepigazzini) [reply MetaData/data/Era2017_RR-31Mar2018_v1/datasets_3.json:1]: This is fine, it's already fix in the upstream branch the change will go unnoticed

File: MetaData/data/cross_sections.json (hunk 4 above)
Comment (simonepigazzini) [reply MetaData/data/cross_sections.json:65]: This we can discuss

File: Systematics/python/flashggDiPhotonSystematics2018_cfi.py (hunk 5 above)
Comment (simonepigazzini) [reply Systematics/python/flashggDiPhotonSystematics2018_cfi.py:389]: This is fine, this PR is dev on t&p and for that you need the latest corrections

File: MetaData/data/cross_sections.json (hunk 4 above)
Comment (simonepigazzini) [reply MetaData/data/cross_sections.json]: Actually, it's fine. The name of the sample was changed upstream, so it's a minor adaptation.
